### DISCUSSION TITLE
is tauri apps easy to decompile like electronjs?

### DISCUSSION THREAD

**Discussion by Nisthar** [tauri-apps/tauri#3204] [Q&A] [answered]
is tauri apps easy to decompile like electron apps?

**Comment by Nisthar** [comment, marked as answer]
Ok its answered here https://tauri.studio/en/docs/about/security/ Thanks

**Comment by wujian752** [comment]
I found it's possible to get the HTML, CSS, JS and other assets from a tauri app.

https://infosecwriteups.com/reverse-engineering-a-native-desktop-application-tauri-app-5a2d92772da5

**Reply by derekdon** [reply]
Wonder if anything has changed in V2 in relation to that post. I believe the authors targeted V1.2.

**Reply by FabianLars** [reply]
Maybe some details if at all but the general gist is the same
